integration-test/src/test/java/com/datastrato/gravitino/integration/test/web/ui/CatalogsPageTest.java

```diff
@@ -108,69 +188,58 @@ public void testViewTabMetalakeDetails() throws InterruptedException {
   }
 
   @Test
-  @Order(4)
+  @Order(7)
   public void testViewCatalogDetails() throws InterruptedException {
+    catalogsPage.createTable(gravitinoUri, metalakeName, catalogName, schemaName);
     catalogsPage.clickViewCatalogBtn(catalogName);
-    String hiveMetastoreUris =
-        String.format(
-            "thrift://%s:%d",
-            containerSuite.getHiveContainer().getContainerIpAddress(),
-            HiveContainer.HIVE_METASTORE_PORT);
-    Assertions.assertTrue(catalogsPage.verifyShowCatalogDetails(catalogName, hiveMetastoreUris));
+    Assertions.assertTrue(catalogsPage.verifyShowCatalogDetails(catalogName, hiveMetastoreUri));
   }
 
   @Test
-  @Order(5)
+  @Order(8)
   public void testEditCatalog() throws InterruptedException {
+
     catalogsPage.clickEditCatalogBtn(catalogName);
     catalogsPage.setCatalogNameField(modifiedCatalogName);
     clickAndWait(catalogsPage.handleSubmitCatalogBtn);
     Assertions.assertTrue(catalogsPage.verifyEditedCatalog(modifiedCatalogName));
   }
 
   @Test
-  @Order(6)
+  @Order(9)
   public void testClickCatalogLink() {
     catalogsPage.clickCatalogLink(metalakeName, modifiedCatalogName);
```

Comment:
Could you pull down the latest commits which including catalog type version, merge the conflicting code and test it again? @ch3yne

Reply:
Tested.